protected/views/patient/_patient_gp.php

```diff
@@ -42,6 +42,14 @@
 			</div>
 		</div>
 		<?php } ?>
+    <div class="row data-row">
+      <div class="large-4 column">
+        <div class="data-label">Practice Role:</div>
```

Comment:
This should be Practitioner Role instead of Practice Role.

Reply:
fixed